inferno/io/transform/volume.py

```diff
@@ -24,10 +27,13 @@ def volume_function(self, volume):
 
 
 class RandomRot3D(Transform):
-    def __init__(self, rot_range, p=0.125,  only_one=True, **super_kwargs):
+    def __init__(self, rot_range, p=0.125, reshape=False, order=0, mode='nearest', **super_kwargs):
```

Comment:
Did the `only_one` attribute do something before?

Reply:
No, I have no clue what it means and what it did before. Check the [original code](https://github.com/inferno-pytorch/inferno/blob/4e25cbb347aa460c0b16216454168310b8f21c87/inferno/io/transform/volume.py#L27)

Reply:
Indeed, then we can safely get rid of it.